apache/application.wsgi.mako

```diff
@@ -0,0 +1,21 @@
+import site
+import sys
+import re
+
+site.addsitedir("${python_path}")
```

Comment:
For example, should I remove `python_path` from `WSGIDaemonProcess` in `apache/wsgi.conf`?